web.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <system.webServer>
+    <handlers>
+      <add name="httpPlatformHandler" path="*" verb="*" modules="httpPlatformHandler" resourceType="Unspecified" />
+    </handlers>
+    <httpPlatform processPath="%JAVA_HOME%\bin\java.exe"
+                  arguments="-Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\send-letter-service-1.1.0.jar&quot;">
```

Comment:
update version in jar to match actual service version (`0.0.1` atm - see build.gradle)

Reply:
sure will update the version